### ISSUE TITLE
Cocos2D-iPhone 2.0 support without special gles20 branch.

### ISSUE THREAD

**Issue by psineur** [cocos2d/cocos2d-iphone-extensions#100]
Subj. See #99

Got TODO list from this:

```
git diff --color --stat develop..gles20

   Extensions/CCScrollLayer/CCScrollLayer.m           |   97 +-
   .../FilesDownloaderTest/FilesDownloaderTestLayer.m |    9 +-
   Tests/FilesDownloaderTest/iTraceurProgressBar.m    |   33 +-
   Extensions/CCLayerPanZoom/CCLayerPanZoom.m         |   23 +-

   Extensions/CCMenuAdvanced/CCMenuAdvanced.m         |    7 +-
   Tests/CCMenuAdvancedTest/CCMenuAdvancedTest.m      |   20 +-

   Extensions/CCSlider/CCSlider.h                     |    2 +-
   Extensions/CCSlider/CCSlider.m                     |    5 +-

   Extensions/CCVideoPlayer/Mac/CCVideoPlayerImplMac.m       |   10 +-

   Tests/CCMenuItemSpriteIndependentTest/CCMenuItemSpriteIndependentTestLayer.m         |    8 +-
   Tests/TMXGeneratorTest/TMXGeneratorTestLayer.m     |    3 +-

   Tests/SharedResources/fps_images-hd.png            |  Bin 0 -> 23443 bytes
   Tests/SharedResources/fps_images-ipadhd.png        |  Bin 0 -> 23443 bytes
   Tests/SharedResources/fps_images.png               |  Bin 6203 -> 6953 bytes

   Tests/SharedSources/RootViewController.h           |   34 -
   Tests/SharedSources/RootViewController.m           |  196 ---
   Tests/SharedSources/cocos2d_extensions_iosAppDelegate.h            |   16 +-
   Tests/SharedSources/cocos2d_extensions_iosAppDelegate.m            |  214 ++--
   Tests/SharedSources/cocos2d_extensions_macAppDelegate.h            |    4 +-
   Tests/SharedSources/cocos2d_extensions_macAppDelegate.m            |   17 +-
   Tests/SharedSources/main.m                         |    2 +-

   Extensions/CCBigImage/CCBigImage.m                 |   11 +-
   Extensions/CCBigImage/CCTextureCache+CCBigImageExtensions.m          |   28 +-
   Tests/CCBigImageTest/CCBigImageTestLayer.m         |    3 +-

   cocos2d                                            |    2 +-
   cocos2d-extensions-ios.xcodeproj/project.pbxproj   | 1784 +++++++++++---------
   cocos2d-extensions-mac.xcodeproj/project.pbxproj   | 1663 +++++++++++--------
```

**Comment by psineur** [comment]
Done in e19a6a2498b3e7ba2d12cba7c658a61502cb38db.

Tested with 2.x both on Mac & iPhone 4S with iOS 5.1
**Must be tested on 1.x and this issue can be closed.**

Removed tag "own feature branch"
Added tag to "develop branch"

**Comment by psineur** [comment]
Closed in v0.2.1
